src/content/experimental/inline-loading/inline-loading-demo-button.js

```diff
@@ -0,0 +1,64 @@
+import mixin from 'carbon-components/es/globals/js/misc/mixin';
```

Comment:
I don't think putting file here is needed from what I read https://github.com/carbon-design-system/carbon-website/blob/6350ab4/src/components/ComponentExample/ComponentExample.js#L7, but please kindly let me know otherwise (e.g. code here is experimental-specific) and let's see together how to make the switch.

Reply:
I don't recall adding this. Sure I would be ok with seeing this.